java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -1484,10 +1492,15 @@ open class KotlinFileExtractor(
                         useFunction<DbCallable>(extractionMethod, getDeclaringTypeArguments(callTarget, drType))
                     }
                 }
-                else
+                else {
                     useFunction<DbCallable>(callTarget)
+                }
 
-            tw.writeCallableBinding(id, methodId)
+            if (methodId == null) {
+                logger.warn("No method to bind call to for raw method access")
```

Comment:
Minor: Can't we use `locId: Label<DbLocation>` as the location of this warning?

Reply:
Do you mean using `warnElement`? If so, we can't get any location information for a `Label`, as the compiler doesn't know what its location is.

We will get the stack of context from https://github.com/github/codeql/blob/main/java/kotlin-extractor/src/main/kotlin/utils/Logger.kt#L133-L137 though

Reply:
I was thinking that we could add an overload of `warn` that takes a `Label<DbLocation>`. But I see that `warnElement` also creates `val locationString = tw.getLocationString(element)`, which we can't do from a label.